initialize-volumes.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+docker volume create data-search
```

Comment:
Could move to `libero/scripts`, and pipe in a list?

Reply:
At the moment the content of this is not very stable, I would expect more volumes and more labels or options to come up before this becomes a library. Also the main goal of `libero/scripts` is reuse, but we don't have another repository that would run this script at the moment.